src/window_identifier/mod.rs

```diff
@@ -254,6 +254,23 @@ impl WindowIdentifier {
     }
 }
 
+#[derive(Default, Debug)]
+pub struct ActivationToken(pub(crate) String);
```

Comment:
the new type needs to be exported in lib.rs & could use some very basic docs + a link the wayland protocol.